Solutions/MCBSTM32F400/DeviceCode/Blockstorage/M29W640FB/M29W640FB_BlConfig.cpp

```diff
@@ -50,12 +50,12 @@ const BlockRange g_M29W640FB_BlockRange1[] =
 const BlockRange g_M29W640FB_BlockRange2[] =
 {
     // ER_DAT
-    { BlockRange::BLOCKTYPE_CODE,   0,  1 },    // 2x64KB = 128KB
+    { BlockRange::BLOCKTYPE_CODE,   0,  2 },    // 2x64KB = 128KB
 #ifdef DEBUG

```

Comment:
The numbers here don't match up with the comments indicating the size (Same for all others below)